.github/config.yml

```diff
@@ -25,3 +25,7 @@ newPRWelcomeComment: >
 # Comment to be posted to on pull requests merged by a first time user
 firstPRMergeComment: >
   Thank you for merging this pull request with us! If you haven't already, in another pull request, please [add yourself](https://github.com/ifmeorg/ifme/wiki/Contributor-Blurb) to our [Contribute](https://www.if-me.org/contribute) page.
+
+# Link for member-bot to use for joining us
+howToJoinLink: >
```

Comment:
I think we can also include some good message like 
> "Thank you for showing interest in joining our community. For further steps visit ...."

Reply:
definitely 👍 although that is a change I will need to make over in the member-cop repo before it can be added here for the app to use. Id like to get this PR in and tested on a live repo before I start shining up member-cop

Reply:
Here is a video. I can @ mention people and use html https://www.screencast.com/t/qcfy3yPFG

Reply:
changing this message text is no longer something that should be done in this PR. Probably best I make a PR in the member-cop repo with the update to use custom messages with the link then we will need a quick PR here to change this file to member-bots new requirements.

Reply:
Nice! That's cool you can @ people :)
Do you have to add the message in the member-cop repo because @ people?

Reply:
correct. Although when I change it I plan to allow people to placeholder in some things like that. Maybe somthing like "welcome [@user] please read [@link] for more info"

Reply:
I wonder if you can concatenate what's in the member-cop repo and what's in this config file.
I don't want to limit your bot to just displaying our welcome message. You could just have `@user` in your member-cop repo and then our welcome message can be in this config?

Reply:
Maybe have a flag in the config too? Like if includeUser is true then concatenate so that the final message is `@user Thanks for expressing your interest. We would love your help with this issue. Please follow these [next steps](https://github.com/ifmeorg/ifme/wiki/Join-Our-Slack) to join our contributor community.`

Reply:
yup its doable 👍 Once this is merged and I know its working I can fine tune those things with a PR in member-cop and 1 here to change the config. Right now member-bot has a generic welcome message that works for any repo, grabs the user from the webhook and the link from the config file concats some things and posts the comment